### PR TITLE
Update redshift-configs.md

### PR DESCRIPTION
**Pull request by Rafalem29**
The existing sort key and dist key examples don't really align with Redshift's best practices. I wanted to propose changes that imply low-cardinality sort keys (which are preferable) and very high cardinality dist keys to ensure parallelism.

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by dataders** [comment]
> changes that imply low-cardinality sort keys (which are preferable) and very high cardinality dist keys to ensure parallelism.

I'm all for this change, but can we go one step further beyond "imply"? Can we not add a sentence that explicitly calls out this best practice?

**Review by dataders — CHANGES_REQUESTED** [review]
> changes that imply low-cardinality sort keys (which are preferable) and very high cardinality dist keys to ensure parallelism.

I'm all for this change, but can we go one step further beyond "imply"? Can we not add a sentence that explicitly calls out this best practice?

**Comment by Rafalem29** [comment]
Oh certainly, I'm in favor of calling this out in the most plain-spoken, clear way possible. I just didn't want to duplicate effort or re-invent (sorry, accidental pun) the AWS Redshift documentation.

**Comment by dataders** [comment]
> Oh certainly, I'm in favor of calling this out in the most plain-spoken, clear way possible. I just didn't want to duplicate effort or re-invent (sorry, accidental pun) the AWS Redshift documentation.

I'll take any language you propose!

**Comment by Rafalem29** [comment]
I think the links to AWS documentation and the modifications to the examples to point to what are, presumably, low-cardinality columns should suffice for now.

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 1]
```diff
@@ -38,31 +38,33 @@ All of these strategies are inherited from dbt-postgres.
 Tables in Amazon Redshift have two powerful optimizations to improve query performance: distkeys and sortkeys. Supplying these values as model-level configurations apply the corresponding settings in the generated `CREATE TABLE` <Term id="ddl" />. Note that these settings will have no effect on models set to `view` or `ephemeral` models.
 
 - `dist` can have a setting of `all`, `even`, `auto`, or the name of a key.
-- `sort` accepts a list of sort keys, for example: `['timestamp', 'userid']`. dbt will build the sort key in the same order the fields are supplied.
+- `sort` accepts a list of sort keys, for example: `['reporting_day', 'category']`. dbt will build the sort key in the same order the fields are supplied.
 - `sort_type` can have a setting of `interleaved` or `compound`. if no setting is specified, sort_type defaults to `compound`.
 
+When working with sort keys, it's highly recommended you follow [Redshift's best practices](https://docs.aws.amazon.com/prescriptive-guidance/latest/query-best-practices-redshift/best-practices-tables.html#sort-keys) on sort key effectiveness and cardinatlity.
```
Comment (matthewshaver) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
When working with sort keys, it's highly recommended you follow [Redshift's best practices](https://docs.aws.amazon.com/prescriptive-guidance/latest/query-best-practices-redshift/best-practices-tables.html#sort-keys) on sort key effectiveness and cardinality. 
```

**Review by matthewshaver — APPROVED** [review]

**Comment by matthewshaver** [comment]
Thank you @Rafalem29 Sorry this took so long to implement

**Comment by matthewshaver** [comment]
Author had addressed previous comments. Went ahead with the merge

**Pull request merged**